frontend/src/components/viewers/MarkdownViewer.test.tsx

```diff
@@ -16,8 +16,10 @@
 
 import * as React from 'react';
 import { mount } from 'enzyme';
-import MarkdownViewer, { MarkdownViewerConfig } from './MarkdownViewer';
+import MarkdownViewer, { MarkdownViewerConfig, MarkdownViewerProps } from './MarkdownViewer';
 import { PlotType } from './Viewer';
+import * as Constants from './Constants';
+import { render, screen, within } from '@testing-library/react';
```

Comment:
within is not used?
You can use vscode "organize imports" feature to autoremove them

Reply:
Thanks for the tip! Removed unused imports